CHANGELOG.md

```diff
@@ -55,6 +55,12 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Vincent Isambart](https://github.com/vincentisambart)
   [#3161](https://github.com/CocoaPods/CocoaPods/issues/3161)
 
+* Added a fix for a bug where some Pods containing Swift frameworks were not
+  being written to the expected directory. Alamofire seems to be a victim in many
```

Comment:
@timrosenblatt: Further investigation revealed that even #3550 doesn't fully resolve the underlying issue. Can you update the changelog entry accordingly and add a reference to this PR? Then we are good to go.